bin/run-prod.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -ex
 
 ./bin/run-common.sh
-exec gunicorn standup.wsgi:application -b 0.0.0.0:${PORT:-8000} --log-file -
+exec newrelic-admin run-program gunicorn standup.wsgi:application -b 0.0.0.0:${PORT:-8000} --log-file -
```

Comment:
In my testing it does not prevent it from running, it just runs. My plan is to make sure stage works w/o issue before adding any settings. We could then configure stage and make sure that works as well before going to prod and configuring that.